docs/apis/management/user_management.yaml

```diff
@@ -116,8 +116,118 @@ paths:
                   message:
                     type: string
                     description: Error message for an unprocessable entity
-
-components: 
+    get:
+      summary: Get users
+      description: Retrieve users data for the specified account.
+      security:
+        - bearerAuth: []
+      tags:
```

Comment:
tags should be same for all commands in this API:

tags:
        - Managing users in bulk